2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,36 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    return (this.volume += 100);
```

Comment:
what happens if `this.volume` is greater than 0 when the `fillUp` method is called? Can you ensure that the bottle capacity of 100 isn't exceeded?

Reply:
Some of these functions don't need to `return` anything because all they do is update the object properties. Can you tell which functions don't need a `return`?